### ISSUE TITLE
fs.truncate missing

### ISSUE THREAD

**Issue by dvlsg** [normalize/mz#33]
It appears [fs.truncate](https://nodejs.org/api/fs.html#fs_fs_truncate_path_len_callback) is missing from the list of fs modules to wrap -- is this intentional? I can confirm that adding `'truncate'` to the `api` array allows me to use it as intended.

Not a huge deal, though, I can just use [ftruncate](https://github.com/normalize/mz/blob/a74ac5ddaf0e7af6dcfed830090fc656f6de7329/fs.js#L12).

**Comment by jonathanong** [comment]
is it new? i don't think it was intentional. PR welcomed!

**Comment by dvlsg** [comment]
To be honest, I have no idea! The only reason I stumbled across it was because I was trying to use a `FileDescriptor` opened with an `a` flag on windows to run `fs.ftruncate`, which apparently always results in an [EPERM error](https://github.com/nodejs/node/issues/3177). Figured I'd just use `fs.truncate(path)` and let node handle opening an fd for me, and I saw it was missing. :)

Of course, I'll put a quick PR together.
